integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedEmbeddedBaseIT.java

```diff
@@ -1617,7 +1822,480 @@ class EntityC {
 						.failure( "Cyclic @IndexedEmbedded recursion starting from type '" + Model.EntityB.class.getName() + "'",
 								"Path starting from that type and ending with a cycle: 'c.b.c.'",
 								"A type cannot declare an unrestricted @IndexedEmbedded to itself, even indirectly",
-								"To break the cycle, you should consider adding filters to your @IndexedEmbedded: includePaths, includeDepth, ..." )
+								"To break the cycle, you should consider adding filters to your @IndexedEmbedded: includePaths, includeDepth, excludePaths, ..." )
+				);
+	}
+
+	@Test
+	public void parentIncludeChildExclude() {
+		class Model {
+			@Indexed(index = INDEX_NAME)
+			class IndexedEntity {
+				@DocumentId
+				Integer id;
+				@KeywordField
+				String indexedEntityString;
+
+				@IndexedEmbedded(includePaths = { "includedString", "subIncluded.subSubIncluded.subSubIncludedString" })
+				Included included;
+
+				public IndexedEntity(Integer id, String indexedEntityString, Included included) {
+					this.id = id;
+					this.indexedEntityString = indexedEntityString;
+					this.included = included;
+				}
+			}
+
+			class Included {
+				@KeywordField
+				String includedString;
+
+				@IndexedEmbedded
+				SubIncluded subIncluded;
+
+				public Included(String includedString, SubIncluded subIncluded) {
+					this.includedString = includedString;
+					this.subIncluded = subIncluded;
+				}
+			}
+
+			class SubIncluded {
+				@KeywordField
+				String subIncludedString;
+
+				@IndexedEmbedded(excludePaths = "subSubIncludedString")
+				SubSubIncluded subSubIncluded;
+
+				public SubIncluded(String subIncludedString) {
+					this.subIncludedString = subIncludedString;
+				}
+			}
+
+			class SubSubIncluded {
+				@KeywordField
+				String subSubIncludedString;
+
+				public SubSubIncluded(String subSubIncludedString) {
+					this.subSubIncludedString = subSubIncludedString;
+				}
+			}
+		}
+
+		assertThatThrownBy( () -> setupHelper.start()
+				.withAnnotatedEntityTypes( Model.IndexedEntity.class )
+				.setup() )
+				.isInstanceOf( SearchException.class )
+				.satisfies( FailureReportUtils.hasFailureReport()
+						.typeContext( Model.IndexedEntity.class.getName() )
+						.pathContext( ".included<no value extractors>.subIncluded<no value extractors>.subSubIncluded" )
+						.failure(
+								"Found conflicting include/exclude paths configuration.",
+								"Included paths are: '[]', excluded paths are: '[subSubIncludedString]'.",
+								"To resolve the conflict either change the configuration of the current property or look at `includePaths` or `excludePaths` of the first node of paths in '[included.subIncluded.subSubIncluded.subSubIncludedString]'."
+						)
```

Comment:
I'd expect this test to just raise the same error as in `error_excludePaths_nonMatched`, because `subIncluded.subSubIncluded.subSubIncludedString` simply doesn't exist... Why do we need a different check?

If you need more arguments:

1. This message says there's a conflict between `includePaths` and `excludePaths`, but also says included paths are empty... ?
2. This message recommends to fix the problem in the child, while the problem really is in the parent which tries to reference a path that doesn't match anything.